src/MaxText/input_pipeline/_grain_tokenizer.py

```diff
@@ -28,30 +28,26 @@ class TokenizeAndTrim(grain.MapTransform):
   """Tokenize and trim features to sequence length."""
 
   # pylint: disable=attribute-defined-outside-init
-  feature_names: str | Sequence[str]
```

Comment:
Let's keep TokenizeAndTrim as it was to allow supporting multiple columns, I think it's needed for the DPO support (https://github.com/AI-Hypercomputer/maxtext/pull/947)

Reply:
You're right. I will roll back TokenizeAndTrim and TokenizeAndChunk can consume multiple columns.